src/scm/github.js

```diff
@@ -411,18 +406,18 @@ class Github {
         data: JSON.stringify(payload)
       });
     })
-    .then((response) => {
+    .then(response => {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/leonhartX/gas-github/blob/40518723e5d56dd5d2636659212a608b09d1f16a/src/scm/github.js#L409"></span>‘response’ is defined but never used.</td>
    </tr>
  </tbody>
</table>